assets/css/detours/_detour_modal.scss

```diff
@@ -1,11 +1,11 @@
 .c-detour-modal {
   inset: 0;
   position: fixed;
-  transition: all 0.3s;
   z-index: map-get($z-page-layout-context, "detour-modal");
 }
 
-.c-detour-modal__hidden {
+.c-detour-modal__transition-exit-active {
```

Comment:
The exit transition works perfectly! The enter one doesn't seem to be the same as it was before. I was playing around a little bit with the `...-enter`/`-enter-active`/etc classes, and I couldn't get it to work, and it might not be worth the effort, but I just wanted to flag that.

Reply:
Turns out in this case it's actually `appear` and `appear-active` - see ef10b0dc.